client/src/pages/DemoUser/DemoUser.tsx

```diff
@@ -0,0 +1,36 @@
+import Button from '@material-ui/core/Button';
+import login from '../../helpers/APICalls/login';
+import { useAuth } from '../../context/useAuthContext';
+import useStyles from './useStyles';
+
+export default function Demo(): JSX.Element {
```

Comment:
```suggestion
export default function DemoUser(): JSX.Element {
```
Just to be consistent with the file name and the usage 👍🏼